data/src/test/java/org/apache/iceberg/data/TestLocalScan.java

```diff
@@ -60,15 +62,12 @@
 import org.apache.iceberg.types.Types;
 import org.apache.iceberg.util.DateTimeUtil;
 import org.assertj.core.api.Assertions;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-
-@RunWith(Parameterized.class)
```

Comment:
this should be replaced with `@ExtendWith(ParameterizedTestExtension.class)`. Please check other tests that use this on how to parameterize a JUnit5 test at the class level

Reply:
@igoradulian this hasn't been addressed yet